extensions/amp-story-interactive/0.1/amp-story-interactive-img.css

```diff
@@ -13,3 +13,58 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+.i-amphtml-story-interactive-img-container {
+  --i-amphtml-interactive-img-option-size: 7.5em !important;
+  --i-amphtml-interactive-img-option-margin: 1em !important;
+
+  border-radius: 3.75em 3.75em 4.375em 4.375em !important;
+}
+
+.i-amphtml-story-interactive-img-option-container {
+  background: var(
+    --i-amphtml-interactive-options-chip-background-color
+  ) !important;
+  display: flex !important;
+  justify-content: center !important;
+  border-radius: 4.375em !important;
+  overflow: hidden !important;
+  box-shadow: var(--i-amphtml-interactive-component-shadow) !important;
+  flex-wrap: wrap !important;
+}
+
+.i-amphtml-story-interactive-img-option {
+  width: var(--i-amphtml-interactive-img-option-size) !important;
+  height: var(--i-amphtml-interactive-img-option-size) !important;
+  position: relative !important;
+  margin-top: var(--i-amphtml-interactive-img-option-margin) !important;
+  margin-right: var(--i-amphtml-interactive-img-option-margin) !important;
+  padding: 0 !important;
+  background: none !important;
+  border: none !important;
+  font-size: inherit !important;
+}
+
+.i-amphtml-story-interactive-img-option:nth-child(odd) {
+  margin-left: var(--i-amphtml-interactive-img-option-margin) !important;
+}
+
+.i-amphtml-story-interactive-img-option:last-child {
+  margin-bottom: var(--i-amphtml-interactive-img-option-margin) !important;
+}
+
+.i-amphtml-story-interactive-img-option:nth-child(3):last-child {
+  margin-top: 0 !important;
+}
```

Comment:
A pattern I just discovered for `rtl` support spacing in flex containers.
Check out this article for more context:
https://web.dev/learn/css/spacing/#grid-and-flexbox

Screen shot of how this could be applied:
<img width="906" alt="Screen Shot 2021-07-02 at 10 02 42 AM" src="https://user-images.githubusercontent.com/3860311/124286287-c7a55300-db1c-11eb-8cde-0c456eb7a3ba.png">

Reply:
Love the idea, but do you think it has enough support? https://caniuse.com/?search=gap

Reply:
Good call. I don't think it has enough coverage for our [supported browsers](https://amp.dev/support/faq/supported-browsers/).
Please disregard, but keep in mind for the future when it gains support!